src/main/resources/edu/cornell/kfs/tax/batch/Default1042STaxOutputDefinition.xml

```diff
@@ -128,28 +132,30 @@
+        <field name="D18" length="25" type="BLANK" /><!-- D18 :: Country Name (NOTE: Should be non-blank if D17 Country Code is not given) -->
+        <field name="D19" length="10" type="DERIVED" value="endDate" /><!-- D19 :: Cycle Date (Payment Date) -->
+        <field name="D20" length="10" type="STATIC" value="KFS" /><!-- D20 :: Record Source -->
         <field name="D21" length="1" type="STATIC" value="A" /><!-- D21 :: Record Status (F=Forecast, A=Actual, O=Out of System, R=Refund) -->
         <field name="D22" length="1" type="STATIC" value="Y" /><!-- D22 :: Payment Type (Y=Year-to-date, C=Cycle only) -->
-        <field name="D23" length="4" type="BLANK" /><!-- D23 :: NQI/FLW-THR ID -->
-        <field name="D24" length="40" type="BLANK" /><!-- D24 :: NQI/FLW-THR Name Line 1 -->
-        <field name="D25" length="4" type="BLANK" /><!-- D25 :: Payer ID -->
-        <field name="D26" length="40" type="BLANK" /><!-- D26 :: Payer Name -->
-        <field name="D27" length="4" type="BLANK" /><!-- D27 :: Primary Withholding Agent's ID -->
-        <field name="D28" length="40" type="BLANK" /><!-- D28 :: Primary Withholding Agent's Name -->
-        <field name="D29" length="10" type="STATIC" value="KFS" /><!-- D29 :: Record Source -->
-        <field name="D30" length="20" type="BLANK" /><!-- D30 :: Campus -->
-        <field name="D31" length="9" type="DERIVED" value="ein" /><!-- D31 :: EIN -->
+        <field name="D23" length="3" type="BLANK" /><!-- D23 :: Number of Days in Cycle (Use to project end dates) -->
+        <field name="D24" length="10" type="DERIVED" value="endDate" /><!-- D24 :: Last Date of Earnings (Last Date for which payments cover. MM/DD/YYYY - (M-Month, D-Day, Y-Year) -->
+        <field name="D25" length="8" type="BLANK" /><!-- D25 :: Constant Additional Tax (Wants Decimal) -->
+        <field name="D26" length="20" type="BLANK" /><!-- D26 :: Campus -->
+        <field name="D27" length="9" type="DERIVED" value="ein" /><!-- D27 :: EIN -->
+        <field name="D28" length="4" type="BLANK" /><!-- D28 :: Intermediary/FLW-THR ID -->
+        <field name="D29" length="40" type="BLANK" /><!-- D29 :: Intermediary/FLW-THR Name Line 1 -->
+        <field name="D30" length="4" type="BLANK" /><!-- D30 :: Other Payer ID -->
+        <field name="D31" length="40" type="BLANK" /><!-- D31 :: Other Payer Name -->
+        <field name="D32" length="4" type="BLANK" /><!-- D32 :: Primary Withholding Agent's ID -->
+        <field name="D33" length="40" type="BLANK" /><!-- D33 :: Primary Withholding Agent's Name -->
         <!--
-            ==== NOTE: The current spec does not have any fields with names D32 through D96. ====
+            ==== NOTE: The current spec does not have any fields with names D34 through D96. ====
          -->
         <field name="D97" length="25" type="BLANK" /><!-- D97 :: Filler -->

```

Comment:
Another spot where it's unclear whether manually including spaces is still needed; same goes for D99 below.